Core/Src/spi_command_handler.c

```diff
@@ -243,264 +243,6 @@ void spi_transfer_image() {
     static uint8_t image_data[IRIS_IMAGE_TRANSFER_BLOCK_SIZE];
     static uint16_t num_transfers;
 
-    uint8_t image_data_buffer[4574] = {
```

Comment:
That's what I get for reading one commit at a time LOL

Reply:
Sorry 😅 I should have cleaned up my history lol